filesystem/src/main/java/org/frankframework/filesystem/FileSystemUtils.java

```diff
@@ -320,6 +319,14 @@ public static <F, FS extends IBasicFileSystem<F>> void getFileInfo(FS fileSystem
 					log.warn("cannot get canonicalName for file [{}]", name, e);
 					file.addAttribute("canonicalName", name);
 				}
+				// Add class of item: file, folder or ask the file system
```

Comment:
Mijn opmerking was eigenlijk vooral over deze IF, volgens mij kan je dat altijd gewoon vragen aan het FS.
En dan kan de parameter weg.

Reply:
Ja, het kan inderdaad, waardoor hij dit altijd vraagt aan het Filesystem. Ik vond het onnodig om dit per file aan het FS te vragen als het toch al duidelijk is, door het type mee te geven. Als de lijsten groot zijn, heeft dit misschien wel impact. Maar goed, dan mag je het achteraf deze optimalisatie weer toevoegen.